scripts/start-gitlab.sh

```diff
@@ -1,16 +1,31 @@
 #!/bin/bash -e
-if [[ $MAKE_TARGET != "testacc" ]]; then echo "not starting gitlab!"; exit 0; fi
+test "$MAKE_TARGET" == "testacc" || { echo "not starting gitlab!"; exit 0; }
 echo "Starting gitlab container..."
-docker run -d -e GITLAB_ROOT_PASSWORD=adminadmin --rm -p 127.0.0.1:8080:80 --name gitlab gitlab/gitlab-ce
+docker run -d --rm --name gitlab \
+  -e GITLAB_ROOT_PASSWORD=adminadmin \
```

Comment:
Do you want this as a variable as well?

Reply:
It could potentially be. For the sake of automated tests it's not even necessary to set the root password since we're not even accessing the UI, but it might be useful if someone wasn't deleting the test resources and was trying to log in afterwards to view them.